yt_dlp/extractor/tv8.py

```diff
@@ -0,0 +1,71 @@
+
+        return {
+            'id': 'tv8',
+            'title': traverse_obj(streaming, ('info', 'title', 'text')),
+            'description': traverse_obj(streaming, ('info', 'description', 'html')),
+            'is_live': True,
+            'formats': self._extract_m3u8_formats(livestream['streaming_url'], 'tv8'),
+        }
+
+
+class TV8PlaylistIE(InfoExtractor):
+    IE_NAME = 'TV8Playlist'
+    IE_DESC = 'TV8 Playlist'
+    _VALID_URL = r'https?://(?:www\.)?tv8\.it/(?!video)(?P<category>[^/]+)/(?P<id>[^/#?]+)'
+    _TESTS = [{
+        'url': 'https://tv8.it/intrattenimento/tv8-gialappas-night',
+        'playlist_mincount': 32,
+        'info_dict': {
+            'id': 'tv8-gialappas-night',
+            'title': 'Tv8 Gialappa\'s Night',
+            'description': 'Tv8 Gialappa\'s Night vi aspetta tutti i mercoledì di UEFA Champions League, in diretta su Tv8. La Gialappa\'s Band si divertirà a commentare con ironia e sarcasmo quello che succede nel mondo del calcio.',
+        },
+    }, {
+        'url': 'https://tv8.it/sport/uefa-europa-league',
+        'playlist_mincount': 11,
+        'info_dict': {
+            'id': 'uefa-europa-league',
+            'title': 'UEFA Europa League',
+            'description': 'Su Tv8 torna il grande calcio europeo con la UEFA Europa League 2024-2025. La competizione è ufficialmente partita lo scorso 11 luglio con il primo turno di qualificazione e proseguirà fino al 21 maggio 2025, data in cui si disputerà la finale all\'Estadio de San Mamés di Bilbao. Per questa edizione ci saranno delle novità, la fase a gironi infatti è stata sostituita da un\'unica fase a campionato con 36 squadre. Ogni club affronterà 8 squadre diverse. Le prime otto classificate staccheranno direttamente il biglietto per gli ottavi di finale, le squadre che si piazzeranno tra la nona e la ventiquattresima posizione invece accederanno alla fase a elimazione diretta solamente tramite gli spareggi.',
+        },
+    }]
+
+    def _real_extract(self, url):
+        video_id, category = self._match_valid_url(url).group('id', 'category')
+        data = self._search_nextjs_data(self._download_webpage(f'https://tv8.it/{category}/{video_id}', video_id), video_id)['props']['pageProps']['data']
+
+        return self.playlist_result([{
+            '_type': 'url_transparent',
+            'ie_key': 'TV8It',
+            'url': f"https://tv8.it{c['href']}",
+            'title': traverse_obj(c, ('title', 'typography', 'text')),
+            'thumbnail': traverse_obj(c, ('image', 'src')),
+            'description': traverse_obj(c, ('extraData', 'videoDesc')),
+            'id': str_or_none(traverse_obj(c, ('extraData', 'asset_id'))),
+        } for c in data['lastContent']['cards']], video_id, traverse_obj(data, ('card', 'desktop', 'title', 'text')), re.sub(r'<.*?>', '', traverse_obj(data, ('card', 'desktop', 'description', 'html'))).strip())
```

Comment:
Why should this be url_transparent?
I would have written this like so:
```suggestion
        entries = [self.url_result(
            urljoin('https://tv8.it', card['href']), ie=TV8ItIE,
                **traverse_obj(card, {
                    'id': ('extraData', 'asset_id', {str}),
                    'title': ('title', 'typography', 'text', {str}),
                    'description': ('extraData', 'extraData', {str}),
                    'thumbnail': ('image', 'src', {url_or_none}),
                }))
            for card in traverse_obj(data, ('lastContent', 'cards', lambda _, v: v['href']))]

        return self.playlist_result(entries, playlist_id,
                **traverse_obj(data, ('card', 'desktop', {
                    'title': ('title', 'text', {str}),
                    'description': ('description', 'html', {clean_html}),
                    'thumbnail': ('image', 'src', {url_or_none}),
                })))
```

Reply:
I saw other extractors are using url_transparent so I did the same. Doesn't calling url_result imply that the playlist extraction will be forced to wait for every video extraction?

Reply:
url_transparent should be used when the info extracted is more precise than the one at the target url https://github.com/yt-dlp/yt-dlp/blob/master/yt_dlp/extractor/common.py#L508

So imagine a site where you embed cloudflare streams, but want to retain the original information.